_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -0,0 +1,227 @@
+<html lang="en">
+	<head>
+		<title>ACT Rules Format 1.0 - Abstract</title>
+	</head>
+	<body>
+		<p>
+			The Accessibility Conformance Testing (ACT) Rules Format 1.0 defines a format for writing accessibility test
+			rules. These test rules can be used for developing automated testing tools and manual testing methodologies. It
+			provides a common format that allows any party involved in accessibility testing to document and share their
+			testing procedures in a robust and understandable manner. This enables transparency and harmonization of testing
+			methods, including methods implemented by accessibility test tools.
+		</p>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+This page has a [default language][default page language] of English because most words are English. The [default page language][] matches the language indicated in the `lang` attribute.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>Gelukkig</title>
+	</head>
+	<body>
+		<p>The Dutch word "gelukkig" has no equivalent in English.</p>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This page has a [default language][default page language] of English, which is not the language indicated in the `lang` attribute.
+
+```html
+<!DOCTYPE html>
+<html lang="da">
+	<head>
+		<title>ACT Rules Format 1.0 - Abstract</title>
+	</head>
+	<body>
+		<p>
+			The Accessibility Conformance Testing (ACT) Rules Format 1.0 defines a format for writing accessibility test
+			rules. These test rules can be used for developing automated testing tools and manual testing methodologies. It
+			provides a common format that allows any party involved in accessibility testing to document and share their
+			testing procedures in a robust and understandable manner. This enables transparency and harmonization of testing
+			methods, including methods implemented by accessibility test tools.
+		</p>
+	</body>
+</html>
+```
+
+#### Failed Example 2
+
+This page has a [default language][default page language] of English because most words are English. This does not match the language indicated in the `lang` attribute.
+
+```html
+<!DOCTYPE html>
+<html lang="nl">
+	<head>
+		<title>Gelukkig</title>
+	</head>
+	<body>
+		<p>The Dutch word "gelukkig" has no equivalent in English.</p>
+	</body>
+</html>
+```
+
+#### Failed Example 3
+
+This page has a [default language][default page language] of Dutch because all English text is wrapped in a `p` element with a `lang` attribute set to `en`. Dutch ("nl") is not set as the language of the page by the `lang` attribute on the `html` element.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>Met de kippen op stok</title>
+	</head>
+	<body>
+		<blockquote>
+			<p>"Hij ging met de kippen op stok"</p>
+		</blockquote>
+		<p lang="en">
+			This Dutch phrase literally translates into "He went to roost with the chickens", but it means that he went to bed
+			early.
+		</p>
+	</body>
+</html>
+```
+
+### Inapplicable
```

Comment:
Missing inapplicable examples for:
* invalid lang attribute
* not a top-level browsing context
* not a text/html content type.

Reply:
The other lang rules don't have this either, for a number of reasons:
- invalid lang would not satisfy the SC.
- not a top-level, I suppose we could add a page with no content, and only an iframe, but that'd give a page without a title. Not sure that's a good thing to put in here.
- not text/html; We can't guarantee these code snippets will be served with the correct MIME type. I don't think it's possible for us to do this at this time.

Reply:
Agreed on the first two points.

> * not text/html; We can't guarantee these code snippets will be served with the correct MIME type. I don't think it's possible for us to do this at this time.

For this one, when we use ` ```xhtml ` instead of ` ```html `, we're adding a different DOCTYPE. I should be enough to serve the example with the correct type, I would say…
We do that in https://act-rules.github.io/rules/5b7ae0#inapplicable-example-4